lib/tasks/contentful.rake

```diff
@@ -80,6 +81,52 @@ namespace :contentful do
 
     puts "URL check complete."
   end
+
+  desc "Update solution with primary category"
+  task update_solution_with_primary_category: :environment do
+    client = Contentful::Management::Client.new(ENV["CONTENTFUL_CMA_TOKEN"])
+    space = client.spaces.find(ENV["CONTENTFUL_SPACE_ID"])
+    environment = space.environments.find("master")
+
+    # Get all solution without a primary category
+    entries = environment.entries.all(content_type: "solution", "fields.primary_category" => nil)
+
+    all_categories = Category.all.map { |cat| [cat.title, cat.id] }.to_h
```

Comment:
It's not particularly important here in a one-off script, but as we're using a modern version of Ruby we don't need the block variable here. This will work:

```ruby
Category.all.map { [it.title, it.id] }.to_h
```